runescape-client/src/main/java/Client.java

```diff
@@ -6224,6 +6224,9 @@ final boolean method1211() {
       garbageValue = "13"
    )
    @Export("openMenu")
+   @Hook(
```

Comment:
Is using the field isMenuOpen not sufficient for this?

Reply:
I could but I'd prefer to only fire the event when the menu is actually opened and not when the field changes from true to false.

Reply:
so, check that it changes to true.

Reply:
oh.. :man_facepalming: